src/Perla/Commands.fs

```diff
@@ -328,6 +344,51 @@ Updated: {package.updatedAt.ToShortDateString()}"""
       return 0
     }
 
+  let runList (options: ListPackagesOptions) =
+    let (|ParseRegex|_|) regex str =
+      let m =
+        Text.RegularExpressions.Regex(regex).Match(str)
+
+      if m.Success then
+        Some(List.tail [ for x in m.Groups -> x.Value ])
+      else
+        None
+
+    taskResult {
+      let! lockFile = Fs.getorCreateLockFile (GetFdsConfigPath())
+
+      let parseUrl url =
+        match url with
+        | ParseRegex @"https://cdn.skypack.dev/pin/(@?[^@]+)@v([\d.]+)"
+                     [ name; version ]
+        | ParseRegex @"https://cdn.jsdelivr.net/npm/(@?[^@]+)@([\d.]+)"
+                     [ name; version ]
+        | ParseRegex @"https://ga.jspm.io/npm:(@?[^@]+)@([\d.]+)"
+                     [ name; version ]
+        | ParseRegex @"https://unpkg.com/(@?[^@]+)@([\d.]+)" [ name; version ] ->
+          Some(name, version)
+        | _ -> None
+
+      match options.format with
+      | HumanReadable ->
+        printfn "Installed packages (alias: packageName@version)"
+        printfn ""
+        for importMap in lockFile.imports do
+          match parseUrl importMap.Value with
+          | Some (name, version) -> printfn $"{importMap.Key}: {name}@{version}"
+          | None -> printfn $"{importMap.Key}: Couldn't parse {importMap.Value}"
+      | PackageJson ->
+        let dependencies =
+          lockFile.imports
+          |> Map.toList
+          |> List.choose (fun (_alias, importMap) -> parseUrl importMap)
+          |> Map.ofList
+
+        JsonSerializer.Serialize({| dependencies = dependencies |}, JsonSerializerOptions(WriteIndented=true))
```

Comment:
Conld you re-use the `Json` internal module for this?
you can find it here
https://github.dev/ThisFunctionalTom/Perla/blob/ac352280164ac5be78ba77a134d4b61061d6819c/src/Perla/IO.fs#L45

Reply:
I moved package.json serialization to Json module. Please look if it is OK now.